include/config.h

```diff
@@ -2,10 +2,6 @@
  * File to store main constants
  */
 #pragma once
-#include "hardware/irq.h"
-#include "hardware/uart.h"
-#include "inttypes.h"
```

Comment:
I am surprised this compiled. I will be explicit in defining the bit size of some of the variables so that `inttypes.h` is not required.

Reply:
Just a note that I modified your HX711 Pin definitions to be a bit more verbose:
```
// HX711 Serial Configuration
constexpr uint8_t HX711_CLKPIN = 17;
constexpr uint8_t HX711_DATPIN = 16;
```

Reply:
> I am surprised this compiled. I will be explicit in defining the bit size of some of the variables so that `inttypes.h` is not required.

I am still surprised that your changes work. I remember following [this stackoverflow post](https://stackoverflow.com/questions/17436527/error-uint16-t-undeclared), which says u need either `stdint.h` or `inttypes.h`.

Reply:
Replaced `inttypes` with `stdint` in [`706e499`](https://github.com/Guppy16/pico-tts/pull/2/commits/706e499a712cc298df70f38255d93ae01749520d)